client/src/i18n/en/account.json

```diff
@@ -47,6 +47,10 @@
     "SUBMIT_DELETE":"Delete Account",
     "SUBMIT_UPDATE":"Update Account",
     "TYPE_CHANGE_BLOCKED":"You cannot change the type of a title account with children.",
+    "CONFIRM_HIDE" : "Hiding an account removes the account from all account selection elements.  It will not affect any pending transactions using the account.  Make it hidden?",
+    "CONFIRM_UNHIDE" : "This account is currently hidden. Are you sure you want to reveal it for use?",
+    "CONFIRM_LOCK" : "Locking an account restrict the ability to post transactions using the account.  Are you sure you want to lock it?",
+    "CONFIRM_UNLOCK" : "The account is locked.  Unlocking it will allow transactions to be posted containing this account.  Are you sure you want to unlock it?",
```

Comment:
There are missing french translations.
```js
"CONFIRM_HIDE" : "Masquer un compte supprime le compte de tous les éléments de sélection de compte. Cela n'affectera pas les transactions en attente utilisant le compte. Le rendre caché ?",
"CONFIRM_UNHIDE" : "Ce compte est actuellement caché. Êtes-vous sûr de vouloir le révéler pour l'utiliser ?",
"CONFIRM_LOCK" : "Le verrouillage d'un compte limite la possibilité de publier des transactions en utilisant le compte. Êtes-vous sûr de vouloir le verrouiller ?",
"CONFIRM_UNLOCK" : "Le compte est bloqué. Le débloquer permettra d'afficher des transactions contenant ce compte. Êtes-vous sûr de vouloir le débloquer ?",
```

Reply:
Ah, beautiful.  This is exactly what I needed!